src/lsg/patterns/chrome/index.tsx

```diff
@@ -20,12 +20,15 @@ const StyledChrome = styled.div`
 	display: flex;
 	align-items: center;
 	width: 100%;
-	height: 54px;
+	height: 40px;
 	padding: ${getSpace(SpaceSize.XS)}px ${getSpace(SpaceSize.XXL) * 3}px;
+	border-bottom: 0.5px solid ${colors.grey90.toString()};
```

Comment:
We should not drop on subpixels - this might produce undesirable output on devices with lower pixel density.
This should be guarded with a  media query, like this:

```
border-bottom-width: 1px;
@media screen and (-webkit-min-device-pixel-ratio: 2) {
  border-bottom-width: 0.5px;
}
```